tests/profiler/test_continuous_profiler.py

```diff
@@ -83,8 +73,7 @@ def test_continuous_profiler_invalid_mode(mode, make_options, teardown_profiling
 @pytest.mark.parametrize(
     "make_options",
     [
-        pytest.param(get_client_options(True), id="non-experiment"),
-        pytest.param(get_client_options(False), id="experiment"),
+        pytest.param(get_client_options()),
```

Comment:
[question] I am not familiar with `pytest.param`. Do we still need the parameterization though? It looks like there is only one item we parameterize by, so unless `pytest.param` somehow expands to more parameterizations, it seems we could get rid of the `pytest.mark.parametrize`

Reply:
not worth the effort